Library/Homebrew/cask/pkg.rb

```diff
@@ -110,28 +110,41 @@ def special?(path)
     RMDIR_SH = <<~'BASH'
```

Comment:
Could this be pulled into a separate `.sh` script in `Library/Homebrew/cask/rmdir.sh` or similar so it can get checked with `shellcheck` and get syntax highlighting?